src/reconcile/rec_write.c

```diff
@@ -1660,6 +1660,11 @@ __rec_child_modify(WT_SESSION_IMPL *session,
 			if (F_ISSET(r, WT_REC_EVICT))
 				return (EBUSY);
 
+			if (ref->addr == NULL) {
+				*statep = WT_CHILD_IGNORE;
+				WT_CHILD_RELEASE(session, *hazardp, ref);
+			}
+
```

Comment:
I had to stare at this again, I think we need a comment in `__evict_page_dirty_update()` to flag that lookaside pages may not have addresses? A comment here wouldn't be bad, either.

Reply:
I'm confused why this isn't an existing bug, that is, I'm not sure why this change wasn't needed before today. Couldn't we get here with an empty lookaside page before?

Reply:
This is an existing bug, but because pages with lookaside entries weren't splitting, our existing tests that do checkpoints with lookaside entries always ended up with a non-empty on-disk page image.